omicron_server/api_server.py

```diff
@@ -130,7 +154,14 @@ def revoke_token():
                 username=username_to_delete
             ).first()
             if user is None:
-                abort(404)
+                response = jsonify(
+                    {'error': 'unable to find user with username %s' %
+                              username_to_delete
+                     }
+                )
+                response.status_code = 404
```

Comment:
This is out of scope for this PR. I opened up issue #118 to deal with this problem